obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -4,13 +4,13 @@
 #include "error.hpp"
 #include "shared.hpp"
 
-obs_output_t*  streamingOutput;
-obs_output_t*  recordingOutput;
-obs_encoder_t* audioStreamingEncoder;
-obs_encoder_t* audioRecordingEncoder;
-obs_encoder_t* videoStreamingEncoder;
-obs_encoder_t* videoRecordingEncoder;
-obs_service_t* service;
+obs_output_t*  streamingOutput			= nullptr;
```

Comment:
The spacing here could probably use some alignment.

Reply:
Corrected!